inst/pages/11_taxonomic_information.qmd

```diff
@@ -188,10 +188,10 @@ for data agglomeration by prevalence.
 ```{r}
 altExp(tse, "Species_byPrevalence") <- mergeFeaturesByPrevalence(tse, 
                                                                rank = "Species", 
-                                                               other_label = "Other", 
+                                                               other.label = "Other", 
                                                                prevalence = 5 / 100, 
                                                                detection = 1 / 100, 
-                                                               as_relative = T)
+                                                               as.relative = T)
```

Comment:
Rename function to agglomerateByRank. Use TRUE instead of T. 

T is just variable that has value TRUE by default. You can overwrite it with t <- "test". ---> good practice to use TRUE